### PR TITLE
feat: only show range annotation information when the feature flag is on

### LINKED ISSUE #1750: annotations: small fixes for when range flag is off
range button showing up in overlay regardless of feature flag status
 range text showing up in toolbar regardless of feature flag status

### PR DESCRIPTION
**Pull request by jrenee42**
Closes #1750 

*  hides the range/point toggle when range annotations are off
*  hides the informational text on the annotations bar when range annotations are off
*  adding e2e tests to test this.

### CODE REVIEW

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 57 | :red_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 93 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=master)

File: src/annotations/components/controlBar/AnnotationsControlBar.tsx [hunk 1]
```diff
@@ -29,25 +27,23 @@ export const AnnotationsControlBar: FC = () => {
   // make both textblocks have 'inline' style to get them on the same line; else there is a line break.
   // using two elements to put space between them.
   return (
-    <FlexBox
+    <Toolbar
       testID="annotations-control-bar"
-      justifyContent={JustifyContent.FlexStart}
       margin={ComponentSize.Large}
+      style={{justifyContent: 'center', padding: '0 32px', width: '100%'}}
```
Comment (hoorayimhelping) [inline src/annotations/components/controlBar/AnnotationsControlBar.tsx:33]: how'd you come up with the padding here? are there clockface defaults we can use?

@daviddkkim

File: src/annotations/components/controlBar/AnnotationsControlBar.tsx [hunk 2]
```diff
@@ -29,25 +27,23 @@ export const AnnotationsControlBar: FC = () => {
...
-          text={infoText2}
-          style={{display: 'inline'}}
-        />
-      </FlexBoxChild>
-    </FlexBox>
+      <TextBlock
+        backgroundColor={InfluxColors.Obsidian}
+        textColor={InfluxColors.Mist}
+        text={infoText1}
+        style={{marginRight: 8, paddingRight: 0}}
```
Comment (hoorayimhelping) [inline src/annotations/components/controlBar/AnnotationsControlBar.tsx:39]: same question here with the margin and padding

**Review by hoorayimhelping — APPROVED** [review]
lgtm, but we might need to follow up if the inline styles aren't following best practices

File: src/annotations/components/controlBar/AnnotationsControlBar.tsx (hunk 1 above)
Comment (jrenee42) [reply src/annotations/components/controlBar/AnnotationsControlBar.tsx:33]: i put in something that looks good.  @daviddkkim  will adjust after.

**Pull request merged**